src/Application.php

```diff
@@ -233,6 +235,22 @@ public function getAdapterFactory()
     /**
      * {@inheritdoc}
      */
+    public function renderException($e, $output)
+    {
+        if ($e instanceof UserException) {
+            $this->getHelperSet()->get('gush_style')->getStyle()->error($e->getMessage());
```

Comment:
Maybe some helper methods like `$this->success (...)`, `$this->error (...)` etc. will make some of the code less verbose